packages/barrier/SOURCES/barrier.sgifixes.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 18b242db..b7694414 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -18,9 +18,9 @@
+ cmake_minimum_required (VERSION 3.4)
+ project (barrier C CXX)
+ 
+-option (BARRIER_BUILD_GUI "Build the GUI" ON)
++option (BARRIER_BUILD_GUI "Build the GUI" OFF)
+ option (BARRIER_BUILD_INSTALLER "Build the installer" ON)
+-option (BARRIER_BUILD_TESTS "Build the tests" ON)
++option (BARRIER_BUILD_TESTS "Build the tests" OFF)
```

Comment:
Same comment about options on/off as above

Reply:
This should be resolved now.